packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ProductFilterOptionsDecorator.types.yaml

```diff
@@ -0,0 +1,24 @@
+ProductFilterOptionsDecorator:
+    type: interface
+    decorator: true
+    config:
+        description: "Represents a product filter options"
+        fields:
+            minimalPrice:
+                type: "Money!"
+                description: "Minimal price of products for filtering"
+            maximalPrice:
+                type: "Money!"
+                description: "Maximal price of products for filtering"
```

Comment:
For empty category, this returns 
```
          "minimalPrice": "0.000000",
          "maximalPrice": "0.000000",
```

I'm not sure whether it's not confusing (other fields returns `null`, except the `inStock`). 
Is it intentionally like this?

Reply:
This one is intentional. Do you suggest some change?

Reply:
Not necessary. I just wanted to be sure 👍